netbox/vpn/urls.py

```diff
@@ -1,89 +1,39 @@
 from django.urls import include, path
 
 from utilities.urls import get_model_urls
-from . import views
+from . import views  # noqa F401
```

Comment:
I do feel like these ought to be avoided, if just because they make things less clear.

Reply:
Agreed, but we need to import the views somehow. If not here, we could do it under the `ready()` method for each app.